week-3/stretch/rotate-char.js

```diff
@@ -8,6 +8,74 @@
 
 // Acceptance criteria:
 
+function rotateCharacter(char, shift) {
+  const alphabetLower = [
+    "a",
+    "b",
+    "c",
+    "d",
+    "e",
+    "f",
+    "g",
+    "h",
+    "i",
+    "j",
+    "k",
+    "l",
+    "m",
+    "n",
+    "o",
+    "p",
+    "q",
+    "r",
+    "s",
+    "t",
+    "u",
+    "v",
+    "w",
+    "x",
+    "y",
+    "z",
+  ];
+  const alphabetUpper = [
+    "A",
+    "B",
+    "C",
+    "D",
+    "E",
+    "F",
+    "G",
+    "H",
+    "I",
+    "J",
+    "K",
+    "L",
+    "M",
+    "N",
+    "O",
+    "P",
+    "Q",
+    "R",
+    "S",
+    "T",
+    "U",
+    "V",
+    "W",
+    "X",
+    "Y",
+    "Z",
+  ];
+  if (alphabetLower.includes(char)) {
+    const value = (alphabetLower.indexOf(char) + shift) % 26;
+    return alphabetLower[value];
+  } else if (alphabetUpper.includes(char)) {
+    const value = (alphabetUpper.indexOf(char) + shift) % 26;
+    return alphabetUpper[value];
+  } else {
+    return char;
+  }
```

Comment:
Ahh I see, yes it makes the code more efficient. So that way we will turn the chat in to lower case and then check if it is included in the lower case array. Then shift the char and return upper case or lower case. When there is no matching element (char is not a letter) in the array we will get -1 and return the char itself